src/types.ts

```diff
@@ -158,9 +161,23 @@ export type OwnerType = {
     Salutation_Dear: string
     Owner: TitleOwner
 }
-
+export type NameOnTitlePropertyType = {
+    Name: string
+    Owner: TitleOwner
+    id: number
+    Name_On_Title:lookupField
+    Property:lookupField
```

Comment:
please install `prettier` and `eslint` in your VS Code so it will fix up these type of issues

Reply:
installed